docsteady/spec.py

```diff
@@ -168,6 +168,32 @@ def process_steps(self, test_script):
         return teststeps_sorted
 
 
+def get_lvv_details(key):
+    """
+    get LVV information from Jira
+    :param key: LVV jira Key
+    :return: needed parameter (first stage, only high level req, if available in Jira
+    """
+    lvv = dict()
+    lvv['high_level_req'] = []
+    if key != "":
+        resp = requests.get(
+            Config.ISSUE_URL.format(issue=key),
+            auth=Config.AUTH
+        )
+        if resp.status_code != 200:
+            print("Unable to download")
+            print(resp.text)
+            sys.exit(1)
+        lvv_resp = resp.json()
+        if lvv_resp['fields']['customfield_13515']:
+            raw0 = lvv_resp['fields']['customfield_13515'].split(', [')
+            for entry in raw0:
+                high_req = entry.split('|')
+                lvv['high_level_req'].append(high_req[0].strip('['))
+    return lvv
```

Comment:
Since the only thing added to the dictionary is `high_level_req`, why not just return that as a list?  Is this for future expansion?

Reply:
Yes, for future expansions, but I am not sure this will really be required.